src/Components/Admin/EnvironmentEditor/conclusionModal.js

```diff
@@ -126,13 +135,38 @@ export default function ConclusionModal({
                         placeholder="The link students will see when finishing the escape room."
                     />
                 </div>
+                <div style={{ paddingBottom: 15 }}>
+                    <Typography
+                        style={{
+                            fontSize: 20,
+                            lineHeight: "27px",
+                        }}
+                    >
+                        External Link
+                    </Typography>
+                    <TextField
+                        value={externalLink}
+                        className={classes.textField}
+                        onChange={handleExternalLinkChange}
+                        required
+                        variant="outlined"
+                        multiline
+                        inputProps={{
+                            style: {
+                                padding: "0px 10px",
+                            },
+                        }}
+                        placeholder="The external link students will can go to after the room is done."
```

Comment:
Small typo. Would suggest changing to:
"The feedback form link students can use to submit feedback" 
Because the external_link attribute is only used for feedback forms.